Feature simbad query tap (#2856)
astropy/astroquery: docs/simbad/simbad.rst

A warning about big queries
---------------------------

> The SIMBAD database is largely used and had to limit the rate of incoming queries.

Reviewer: ```suggestion
The SIMBAD database is widely used and has to limit the rate of incoming queries.
```
